### PR TITLE
Team list implemented as a div changed to list(ul), with vertical scrollbar

### LINKED ISSUE #156: Convert team listing to a list with vertical scrollbar
Team list is implemented as a div, it has to be made into a list (ul) with vertical scrollbar

### PR DESCRIPTION
**Pull request by sadrisya**
Fixes #156

Screenshot 2024-11-05 at 2 34 39 PM (2)

### CODE REVIEW

File: app/views/teams/_sub_teams.html.erb [hunk 1]
```diff
@@ -19,9 +19,8 @@
     <% end %>
   </div>
 <% end %>
-
```
Comment (deepakkumarnd) [inline app/views/teams/_sub_teams.html.erb:22]: Do not remove these blank lines. They are kept for idnentifying the blocks

File: app/views/teams/_sub_teams.html.erb [hunk 2]
```diff
@@ -19,9 +19,8 @@
...
-<div id="team-list-desktop" class="flex flex-col gap-4 pt-6">
+<ul class="team-list flex flex-col gap-4 mt-6 overflow-y-auto max-h-[240px]">
```
Comment (deepakkumarnd) [inline app/views/teams/_sub_teams.html.erb:22]: what is the use of team-list class ?, have you tested the code ? If you remove the "id" then adding the new item won't reflect in the list.

**Review by deepakkumarnd — CHANGES_REQUESTED** [review]

File: app/views/teams/_sub_teams.html.erb (hunk 2 above)
Comment (sadrisya) [reply app/views/teams/_sub_teams.html.erb]: It is a typo, even when I tested couldn't find any issue ,so missed it

File: app/views/teams/_sub_teams.html.erb (hunk 1 above)
Comment (sadrisya) [reply app/views/teams/_sub_teams.html.erb:22]: this is happening on auto format while saving

Comment (sadrisya) [reply app/views/teams/_sub_teams.html.erb:22]: Added the Blank line

File: app/views/teams/_sub_teams.html.erb (hunk 2 above)
Comment (sadrisya) [reply app/views/teams/_sub_teams.html.erb]: ID is replaced

**Pull request merged**